Bottom: 3ac1b32e2ea9332a17cdc3fca5594966d1e49519
Top:    dbe49115a5d644d7d7dc686ed8891dca3e055f02
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-09 23:53:29 +0000

Refresh of ensure-we-throttle-redraws-fro

---

diff --git a/src/hid/gtk/gui-trackball.c b/src/hid/gtk/gui-trackball.c
index f5f741a..6a42f59 100644
--- a/src/hid/gtk/gui-trackball.c
+++ b/src/hid/gtk/gui-trackball.c
@@ -268,8 +268,6 @@ ghid_trackball_constructor (GType type,
 static void
 ghid_trackball_finalize (GObject * object)
 {
-//  GhidTrackball *pinout = GHID_TRACKBALL (object);
-
   G_OBJECT_CLASS (ghid_trackball_parent_class)->finalize (object);
 }
 
@@ -290,8 +288,6 @@ static void
 ghid_trackball_set_property (GObject * object, guint property_id,
 				  const GValue * value, GParamSpec * pspec)
 {
-//  GhidTrackball *pinout = GHID_TRACKBALL (object);
-
   switch (property_id)
     {
     default:
@@ -410,7 +406,7 @@ ghid_trackball_get_type ()
 }
 
 
-/*! \brief Convenience function to create a new pinout preview
+/*! \brief Convenience function to create a new trackball widget
  *
  *  \par Function Description
  *  Convenience function which creates a GhidTrackball.
